governance/votes/Executive vote - January 14, 2022.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Template - [Executive Vote] Recover DAI from Optimism Escrow, Changes to Uniswap LP Vault Debt Floor Parameters, Delegate Compensation - January 14, 2022
+summary: Recover DAI from Optimism escrow, Uniswap LP Vault Debt Floor parameter changes, Recognized Delegate compensation distribution for December.
```

Comment:
Maybe do Debt Floor (dust) we're still in the process of pushing people towards other terminology.

Reply:
changed in summary, did you want the title changed as well?

Reply:
Title is fine, I think.